### PR TITLE
Fix missing err check

### PR DESCRIPTION
**Pull request by patrickxb**
r? @maxtaco

### CODE REVIEW

**Comment by patrickxb** [comment]
or @oconnor663

**Comment by maxtaco** [comment]
whoops 👍

File: go/engine/identify2_with_uid.go [hunk 1]
```diff
@@ -196,7 +196,7 @@ func (e *Identify2WithUID) runReturnError(ctx *Context) (err error) {
 		e.G().Log.Debug("- Released singleflight lock")
 	}()
 
-	if e.loadAssertion(); err != nil {
+	if err = e.loadAssertion(); err != nil {
```
Comment (oconnor663) [inline go/engine/identify2_with_uid.go]: There are lints that catch this sort of thing. Does the `if` statement confuse them, or have we not turned them on yet?

**Comment by patrickxb** [comment]
Old code was generating a ton of (unchecked) errors on empty user assertions.  Lots of tests failed.  Most recent commit should fix that.

**Comment by patrickxb** [comment]
two CI errors unrelated.  merging.

**Pull request merged**